pipettes/core/tasks.cpp

```diff
@@ -39,27 +43,36 @@ void pipettes_tasks::start_tasks(
     auto& tasks = pipettes_tasks::get_tasks();
```

Comment:
Are all these pipette tasks in any particular order?

Reply:
Not really. Freertos has its own internal scheduling. If we cared about tasks being executed at a certain time, we would need to modify the priorities.